mysql-test/suite/versioning/r/foreign.result

```diff
@@ -6,6 +9,9 @@ id int unique key
 ) engine innodb;
 create table child(
 parent_id int,
+s bigint unsigned as row start invisible,
+e bigint unsigned as row end invisible,
```

Comment:
Test both combinations: timestamp, trx_id

Use `eval create or replace table (..., sys_start $sys_datatype_expl, sys_end $sys_datatype_exp, ...)`.

Reply:
These tests was originally written for trx_id-based innodb tables.
Before this patch `dict_index_t::vers_start|end` fields was not `0` resulting in an index being `versioned() == true`. Patch fixes that. Foreign check is not implemented for timestamp-based innodb tables. Tests passed by accident, I guess.

Reply:
Please, fix like was said above. The test pass because it does work with timestamp-based except #415. Please, be careful to read comments in file.